start-from-zero/sass-architecture/sass/components/_accordion.sass

```diff
@@ -73,12 +77,17 @@
 
 .details-http
     display: inline-block
-    width: 22%
+    width: 15.783%
```

Comment:
This seems oddly familiar... any particular reason why these styles are not bundled up where overlapping?